aip/client-libraries/4222.md

```diff
@@ -50,6 +50,10 @@ rpc CreateTopic(CreateTopicRequest) {
 }
 ```
 
+The field specified in `field` **must** be a field of type `string` and **must**
+have a path-like value format, resembling a resource name or a HTTP path
+segment. Parsing arbitrary `string` fields is not supported.
```

Comment:
Saying "resembling a HTTP path" might lead the end-user to think that field paths are `/`-separated.

I think below might be a better way to explain this:

The field specified in the `field` field **must** be either 
(1) a name of the field of type `string` in the request message
e.g. `parent`, if the request message has a field named `parent` of type `string`

(2) a path to a field of type `string` in a sub-message of the request message. The path is a series of the names of sub-message fields separated by the `.`. 
E.g. `library.shelf.book` if 
* the request message has a field named `library` of some message type
* the `library` message has a field named `shelf` of some message type
* the `shelf` message has a field named `book` of type `string`.

Reply:
Ok, I've given it another shot. I broke it up into the guidance for format of the `field` value and the guidance for the field that the `field` represents.

WDYT